cmd/commands/root.go

```diff
@@ -0,0 +1,56 @@
+/*
+Copyright 2018 The Rook Authors. All rights reserved.
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package command
+
+import (
+	"github.com/spf13/cobra"
+)
+
+var (
+	KubeConfig           string
+	OperatorNamespace    string
+	CephClusterNamespace string
+)
+
+// rookCmd represents the rook command
+var RootCmd = &cobra.Command{
+	Use:     "rook-ceph",
+	Short:   "kubectl rook-ceph provides common management and troubleshooting tools for Ceph.",
+	Args:    cobra.MinimumNArgs(1),
+	Version: "v0.5.0",
+}
+
+// Execute adds all child commands to the root command and sets flags appropriately.
+// This is called by main.main(). It only needs to happen once to the rootCmd.
+func Execute() {
+	cobra.CheckErr(RootCmd.Execute())
+}
+
+func init() {
+
+	// Here you will define your flags and configuration settings.
```

Comment:
nit: remove the wording that sounds like a template
```suggestion
	// Define your flags and configuration settings.
```

Reply:
done